### ISSUE TITLE
A11y_SQLMigrationControlPlane_MigrateTo AzureSQL_ AssessmentResultsAndRecommendations: Color contrast ratio of the 'Next' button is 4.4:1, which is less than the required one i.e.,4.5:1.

### ISSUE THREAD

**Issue by BhanuPrakashGit** [microsoft/azuredatastudio#20620]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11yMAS;#A11yTCS;#AzureDataStudioextensionAzureSQLMigration-Mac-Sept2022;#SQL Migration Control Plane;#A11ySev2;#WCAG4.3.3;#DesktopApp;#Win11;#ColorContrast;#Benchmark;#Mac

### Environment Details:
Azure Data Studio: Version: 1.39.1
Operating System: Mac Version 12.5.1

### Steps to Reproduce:

1. Install Azure data studio and open
2. Tab till Extension press enter key and install Azure SQL Migration 
3. Tab till Respective server and right click
4. Tab till manage and press enter key
5. Tab till Azure SQL Migration and press enter key
6. Tab till migrate to azure sql and press enter key and Saved session page will open 
7. Select start new session and give next ,Databases for assessment page will open 
8. Select the respective databases and give next 
9. Assessment results and recommendations page will open.

### Actual:
Color contrast ratio of the 'Next' button is 4.4:1, which is less than the required one i.e.,4.5:1.
### Expected:
Color contrast ratio of the 'Next' button should be equal or greater than 4.5:1.
### User Impact:
Low vision users won't be able to see the label present over the text.
![MicrosoftTeams-image (12)](https://user-images.githubusercontent.com/104351566/191017055-7f7f4a4f-f7bd-4c3e-b096-5fc5b6f96389.png)

**Comment by InduPriya1805** [comment]
Githubtags:  #AzureDataStudioextensionAzureSQLMigration-Mac-Sept2022;

**Comment by aasimkhan30** [comment]
@BhanuPrakashGit and @InduPriya1805, the contrast is 4.5:1. Can you please re-verify and close this issue?
<img width="564" alt="image" src="https://user-images.githubusercontent.com/6816294/220279836-6533c9bb-37a7-471b-8a6b-d0df10078f20.png">

**Comment by sspatil123** [comment]
GithubTags:#Closed;Version 1.41.2
![image](https://user-images.githubusercontent.com/85863999/220535553-755cf0bd-f799-44d6-9a1a-0827a5ad82cd.png)

**Comment by fsteffi** [comment]
hi @aasimkhan30  and @alanrenmsft  could you please close this bug as we dont have the permission to do so
